### PR TITLE
Gpl 504 move a container by scanning

### LINKED ISSUE #128: GPL-504 Move a container by scanning [RT 691039]
**User story**
GPL-504 | As a Labwhere user I would like to be able to move a location in labwhere to another location in labwhere, using the scanning in/out function. This is currently limited to pieces of labware.

**Who are the primary contacts for this story**
Danni W
Steve I
Eduardo M

**Acceptance criteria**
To be considered successful the solution must allow:

**Dependencies**
This story is blocked by the following dependencies:
- #<issue_no.>
- sanger/<repo>#<issue_no.>

**Additional context**
From RT 691039. Impacts on Lighthouse sampling strategy

### PR DESCRIPTION
**Pull request by stevieing**
Closes #128

Changes proposed in this pull request:

* Allow users to move locations in bulk from one place to another.
*
* ...

### CODE REVIEW

File: app/forms/move_location_form.rb [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+  def form_name
+    :move_location_form
+  end
+
+  def submit(params)
+    @params = params
+    assign_params
+    assign_attributes
+    if valid?
+      parent_location.children = child_locations
```
Comment (emrojo) [inline app/forms/move_location_form.rb:24]: Ok, just a question, so the idea is that will not add new locations to current list, but overwrite with a new list of locations. Is that right?

Comment (stevieing) [reply app/forms/move_location_form.rb:24]: Good sport. I think the method itself just changes the parent on each location. I will add a test to check that it doesn't overwrite.

Comment (stevieing) [reply app/forms/move_location_form.rb:24]: I have just checked and it does just add children.

File: app/views/move_locations/_form.html.erb [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+      <li>
+        <div><%= f.label :parent_location_barcode, 'Parent location barcode*' %> </div>
+        <div><%= f.text_field :parent_location_barcode %></div>
+      </li>
+      <li>
+        <div><%= f.label :child_location_barcodes, 'Child location barcodes*' %></div>
+        <div class="pr-5"><%= f.text_area :child_location_barcodes %></div>
+      </li>
+      <li>
+          <%= f.submit "Go!", data: {disable_with: "scanning..."} %>
```
Comment (jbeast) [inline app/views/move_locations/_form.html.erb:17]: Love the enthusiasm!

**Pull request merged**